modal/_functions.py

```diff
@@ -1529,6 +1554,8 @@ def _get_metadata(self):
             input_plane_region=self._input_plane_region,
             max_object_size_bytes=self._max_object_size_bytes,
             _experimental_flash_urls=self._experimental_flash_urls,
+            supported_input_formats=self._metadata.supported_input_formats if self._metadata else None,
+            supported_output_formats=self._metadata.supported_output_formats if self._metadata else [],
```

Comment:
Why is the default input format `None` while the output format's default is `[]`?

Reply:
Not intentional, thanks for spotting, fixing. I think it doesn't have any effect in practice though since python-proto will encode both the same.